src/cli/global/list.rs

```diff
@@ -1,9 +1,390 @@
+use crate::global::common::find_package_records;
+use crate::global::project::ParsedEnvironment;
+use crate::global::{EnvironmentName, ExposedName, Project};
 use clap::Parser;
+use fancy_display::FancyDisplay;
+use human_bytes::human_bytes;
+use indexmap::IndexMap;
+use itertools::Itertools;
+use miette::{miette, IntoDiagnostic};
+use pixi_config::{Config, ConfigCli};
+use pixi_consts::consts;
+use pixi_spec::PixiSpec;
+use rattler_conda_types::{PackageName, PackageRecord, PrefixRecord, Version};
+use serde::Serialize;
+use std::io::{stdout, Write};
+use std::str::FromStr;
+use thiserror::__private::AsDisplay;
 
 /// Lists all packages previously installed into a globally accessible location via `pixi global install`.
+///
+/// All environments:
+/// - Yellow: the binaries that are exposed.
+/// - Green: the packages that are explicit dependencies of the environment.
+/// - Blue: the version of the installed package.
+/// - Cyan: the name of the environment.
+///
+/// Per environment:
+/// - Green: packages that are explicitly installed.
```

Comment:
I just saw that this is unfortunately rendered really badly 😢

![grafik](https://github.com/user-attachments/assets/d8307e02-5003-4357-a534-cd3211719d55)

Reply:
Fixed